containers/configs/footer.html

```diff
@@ -0,0 +1 @@
+Issues should be reported to the <a href="https://lists.fedoraproject.org/archives/list/openscanhub@lists.fedoraproject.org/">mailing list</a>.
```

Comment:
Same here.  I would replace `should` with `can`.  I know it is taken from the internal instance but Fedora users may appreciate more freedom :smiley: